content/administration/maintain/on_premise.rst

```diff
@@ -143,26 +132,8 @@ Duplicate a database
 ====================
```

Comment:
This part explains how to duplicate a database for production purpose but most of the time customers will want to duplicate a database to do some testing like on saas or sh. To do so they can check the checkbox neutralize when making a duplicate via <odoo-server>/web/database/manager
This will execute all the neutralize.sql scripts for every installed module. Here we should link to the documentation page about neutralize which I think do not exist (feature introduced in 16: https://github.com/odoo/odoo/pull/102792)